web-console/knikubevirt/vm-details/vm-boot-order/vm-boot-order.md

```diff
@@ -0,0 +1,51 @@
+# VM details - boot order
+
+## Entering edit mode
+![VM details - Overview tab](img/B1-0-0.jpg)
+From the VM details page, Overview tab, the user clicks the pencil icon to edit the boot order.
+
+## Removing source items
+![Modal - default](img/B1-1-0.jpg)
+The modal pops up, displaying the currently listed bootable items by order of boot.
+This order was automatically set on the VM creation. This list displays bootable disks and NICS only.
```

Comment:
Should we clarify here that CD-ROMs can be listed here as well?

Reply:
@jelkosz is this list populated by ALL boot sources by default?

Reply:
I've added CD-ROM in the list that should be included by default on the list